### PR TITLE
Add OpenFigi enricher

### PR DESCRIPTION
**Pull request by jbothma**
- [ ]  update nomenklatura

### CODE REVIEW

File: datasets/_externals/ext_gleif.yml [hunk 1]
```diff
@@ -15,8 +15,6 @@ deploy:
   disk: "20Gi"
   # premium: true
 load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
-coverage:
```
Comment (pudo) [inline datasets/_externals/ext_gleif.yml:18]: Why are we getting rid of this?

File: datasets/_externals/ext_open_figi.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+type: external
+title: OpenFIGI
+entry_point: zavod.runner.enrich:enrich
+full_dataset: openfigi
```
Comment (pudo) [inline datasets/_externals/ext_open_figi.yml:4]: This doesn't apply as long as we don't have a bulk version.

File: datasets/_externals/ext_open_figi.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+disabled: false
+hidden: false
+coverage:
+  frequency: weekly
```
Comment (pudo) [inline datasets/_externals/ext_open_figi.yml:8]: Let's put it on daily for now?

File: datasets/_externals/ext_gleif.yml (hunk 1 above)
Comment (jbothma) [reply datasets/_externals/ext_gleif.yml:18]: duplication https://github.com/opensanctions/opensanctions/pull/458/files/c41bff52b52485267cd62712957922b8d47d0fe2#diff-4a36d4536b29d23bd921ee69a2f61e304cd1d562208442703359fd54ed5d301bL7

Comment (pudo) [reply datasets/_externals/ext_gleif.yml:18]: me moron. me go home

Comment (jbothma) [reply datasets/_externals/ext_gleif.yml:18]: better safe than sorry!

**Pull request merged**